observability/readme.md

```diff
@@ -62,6 +62,14 @@ contributions.
 
 Read more [here](pitfalls.md) to understand what to watch out for while designing and building an observable system.
 
+## Examples of Observability As Code
+
+1. Dashboards as Code- You can create Monitoring Dashboards as Json or XML templates, such templates can be checked into the code repository for source control, any change in the dashboards can be reviewed and automation can be built for enabling the dashboard. [More about how to do this in Azure A](https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-dashboards-create-programmatically).
```

Comment:
Should we also add some reference to grafana dashboards, if any? I am asking about it cause we are mentioning grafana all over the observability section,

Reply:
done